compiler/test/grainfmt/enums.expected.gr

```diff
@@ -31,18 +32,11 @@ enum ParsedRegularExpression {
   REWordBoundary,
   RENotWordBoundary,
   RELiteral(Char),
-  RELiteralString(
-    String
-  ), // <- sequences of literals are flattened into a string
+  RELiteralString(String), // <- sequences of literals are flattened into a string
   REAlts(ParsedRegularExpression, ParsedRegularExpression),
   RESequence(List<ParsedRegularExpression>, Bool), // seq elts, needs backtrack
   REGroup(ParsedRegularExpression, Number), // regex, group ID
-  RERepeat(
-    ParsedRegularExpression,
-    Number,
-    Option<Number>,
-    Bool
-  ), // regex, min, max (None for infinity), true=non-greedy
+  RERepeat(ParsedRegularExpression, Number, Option<Number>, Bool), // regex, min, max (None for infinity), true=non-greedy
```

Comment:
This seems a lot harder to read now.

Reply:
I don't think so... We don't really want the comment to change how it would format if the comment weren't there, and if the comment weren't there, it would be on one line.